test/common/test_type_traits.cpp

```diff
@@ -36,11 +36,11 @@
  */
 
 
-#include <pcl/pcl_macros.h>
 #include <pcl/point_traits.h>
 #include <pcl/point_types.h>
 
 #include <gtest/gtest.h>
+#include <pcl/test/pcl_macros.h>
```

Comment:
Our header grouping strategy, when it is enforced, packs together headers from the same project. In this situation just leave it where it was.